DataLayer/Migrations/Scripts/20190414094415_AddedApplicationUserAndFriendship_Up.sql

```diff
@@ -0,0 +1,74 @@
+﻿DROP TRIGGER IF EXISTS TR_Friendship_InsteadOfInsert;
+
+GO
+
+CREATE TRIGGER TR_Friendship_InsteadOfInsert
+  ON friendships
```

Comment:
Again a reminder, once the table name is changed to Friendship  this trigger will stop working.
Additionally: capitalize table and column names. SQL Server naming convention uses PascalCase almost exclusively outside of T-SQL variable names. And `INSERTED` is a keyword, so all caps.

Reply:
Did everything.